scalableminds/webknossos-wrap
# Adds File.compress and Dataset.compress to Python API

* Adds `File.compress` to Python API
* Adds `Dataset.compress` to Python API with optional parameter to compress all existing files aswell

Depends on #15 

## How to test
* `cd python`
* `python3 setup.py install`
* `python3`:
```
import wkw
import numpy as np

d = wkw.Dataset.create('./tmp', wkw.Header(np.uint8))
a = np.random.uniform(0, 255, (32, 32, 32)).astype(d.header.voxel_type)
d.write((128,128,128), a)
d2 = d.compress('./tmp2', True)
d2.read((128,128,128), (32,32,32))
assert np.all(d2.read((128,128,128), (32,32,32)) == a)
```
* Check file size of `./tmp2/z0/y0/x0.wkw`
